src/ClientData/include/ClientData/TracepointData.h

```diff
@@ -36,36 +38,35 @@ namespace orbit_client_data {
  */
 class TracepointData {
  public:
-  // Assume that the corresponding tracepoint of tracepoint_hash is already in unique_tracepoints_
-  void EmplaceTracepointEvent(uint64_t time, uint64_t tracepoint_hash, uint32_t process_id,
+  // Assume that the corresponding tracepoint of tracepoint_id is already in unique_tracepoints_
+  void EmplaceTracepointEvent(uint64_t timestamp_ns, uint64_t tracepoint_id, uint32_t process_id,
                               uint32_t thread_id, int32_t cpu, bool is_same_pid_as_target);
 
   void ForEachTracepointEventOfThreadInTimeRange(
       uint32_t thread_id, uint64_t min_tick, uint64_t max_tick_exclusive,
-      const std::function<void(const orbit_client_protos::TracepointEventInfo&)>& action) const;
+      const std::function<void(const TracepointEventInfo&)>& action) const;
 
-  void ForEachTracepointEvent(
-      const std::function<void(const orbit_client_protos::TracepointEventInfo&)>& action) const;
+  void ForEachTracepointEvent(const std::function<void(const TracepointEventInfo&)>& action) const;
 
   uint32_t GetNumTracepointEventsForThreadId(uint32_t thread_id) const;
 
-  bool AddUniqueTracepointInfo(uint64_t key, orbit_grpc_protos::TracepointInfo tracepoint);
+  bool AddUniqueTracepointInfo(uint64_t key, TracepointInfo tracepoint);
 
-  [[nodiscard]] orbit_grpc_protos::TracepointInfo GetTracepointInfo(uint64_t hash) const;
+  [[nodiscard]] const TracepointInfo* GetTracepointInfo(uint64_t hash) const;
```

Comment:
The parameter's name should be `tracepoint_id`, right?

Reply:
Done, here and below.